pkg/fmmon/analyze.go

```diff
@@ -0,0 +1,46 @@
+package fmmon
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/goodbuns/fishmon/pkg/adafruitio"
+)
+
+// Analyze analyzes feed information and alerts on any issues.
+func Analyze(feeds *[]adafruitio.Feed, alerts chan string, numFeeds int, minTemp, maxTemp float64, webhookURL string, update bool) {
```

Comment:
This function is a hot mess. Let's pair on this.

Among other things:

1. You probably don't need a channel here. Nothing here is happening _concurrently_. The channel is just being used as a convoluted return mechanism.
2. There's a lot of different things mixed in here: alerting logic (when should I alert?), formatting logic (what should the alert look like?), and "update" logic (I'm still not clear on what this is).